settings.gradle.kts

```diff
@@ -0,0 +1,24 @@
+import java.net.URI
+
+pluginManagement {
+    repositories {
+        google()
+        mavenCentral()
+        gradlePluginPortal()
+    }
+}
+
+@Suppress("UnstableApiUsage")
```

Comment:
What warning is this suppressing?

Reply:
Apparently `repositoriesMode` ([getRepositoriesMode()](https://docs.gradle.org/current/kotlin-dsl/gradle/org.gradle.api.initialization.resolve/-dependency-resolution-management/get-repositories-mode.html)) is marked as incubating. I don't know if the `@Incubating` is about the Kotlin DSL API itself or if the groovy implementation didn't have a way to represent that the API was subject to change.